src/components/Error/Error.scss

```diff
@@ -0,0 +1,3 @@
+.p{
+    color:red;
+}
```

Comment:
This style isn't being used by your component.  Also, it would be better to give it a more descriptive name.